assets/src/edit-story/components/canvas/singleSelectionMoveable.js

```diff
@@ -271,7 +271,10 @@ function SingleSelectionMoveable({
         set(frame.translate);
         return undefined;
       }}
-      onDragEnd={({ target }) => {
+      onDragEnd={({ isDrag, target }) => {
```

Comment:
Should this go for the `multiSelectionMoveable` as well?

Reply:
Done.

Reply:
Actually had to undo that. `onDragGroupEnd` calls `clickHandled` which allows selecting one element in a group, so in this case (multi select) we shouldn't check for isDrag.